docs/releasing.md

```diff
@@ -0,0 +1,12 @@
+# Releasing the collection
+
+To release the collection the following steps are required:
+
+* Create a pull request to update `galaxy.yml` with:
+  * Version bump
+  * Update list of authors
```

Comment:
`git shortlog -e -s | sed -E 's/^\s+[[:digit:]]+\s+(.*)$/  - "\1"/'` can do that

Reply:
(yes, you can't tell `git` not to print the numbers if you want the "summary")